jdaviz/core/loaders/importers/image/image.py

```diff
@@ -202,7 +217,12 @@ def output(self):
 
         return data
 
-    def __call__(self, show_in_viewer=True):
+    def __call__(self, show_in_viewer=True, gwcs_to_fits_sip=None):
+
+        # if 'gwcs_to_fits_sip' is passed in, use it to override
+        # whatever the try_gwcs_to_fits_sip traitlet is set to.
+        gwcs_to_fits_sip = gwcs_to_fits_sip if gwcs_to_fits_sip is not None else self.gwcs_to_fits_sip  # noqa
```

Comment:
I don't think its necessary to support overriding as a kwarg to call (we don't do that for other cases except `show_in_viewer` just to keep things simple), let's just require setting the traitlet.

Reply:
my thought is that the since traitlet will always have a true / false value, so it can't really be passed in as an arg to call (which is what you need to do to set gwcs_to_fits_sip when calling load_data ) unless it's overriding the traitlet value. gwcs_to_fits_sip as an argument to load_data would be meaningless then, i think?

Reply:
When `load_data` calls `load`, all kwargs passed to `load` are passed along through the public API (see `jdaviz.core.helpers.ConfigHelper._load`).  Nothing is passed along to `__call__` unless a user does that manually via `ldr.importer(...)`.